.gitignore

```diff
@@ -1,2 +1 @@
-.vscode
```

Comment:
Undo this please. The settings file is forced, so it works fine as it was. But vscode stores some garbage there that we don't want to track by default.

Alternatively you can use this template: https://www.toptal.com/developers/gitignore/api/visualstudiocode